### ISSUE TITLE
feature request: update to DBUnit 2.7.0

### ISSUE THREAD

**Issue by aklemp** [database-rider/database-rider#111]
Update the underlying dbunit dependency to 2.6.0. It contains several new features and improvements (e.g. support for PostgreSQL data type OID).

**Comment by rmpestano** [comment]
You can already use dbunit 2.6.0 in current version of db-rider, just use maven exclusions and provide dbunit dependency on your project.

Anyway, I've updated dbunit version in db-rider so in next version it should bring v2.6.0

Can you try with v1.5.1-SNAPSHOT of db-rider?

To enable database rider snapshots on your project, [see here](https://github.com/database-rider/database-rider#snapshots).

**Comment by rmpestano** [comment]
Well it needs to be investigated a bit further because dataset comparison with regex is not working on dbunit 2.6.0, I'm getting this error:

```
org.dbunit.dataset.datatype.TypeCastException: Unable to typecast value <regex:\d+> of type <java.lang.String> to BIGINT
	at org.dbunit.dataset.datatype.BigIntegerDataType.typeCast(BigIntegerDataType.java:88)
	at org.dbunit.dataset.datatype.AbstractDataType.compare(AbstractDataType.java:83)
	at org.dbunit.assertion.comparer.value.IsActualEqualToExpectedWithEmptyFailMessageValueComparer.isExpected(IsActualEqualToExpectedWithEmptyFailMessageValueComparer.java:27)
	at org.dbunit.assertion.comparer.value.ValueComparerTemplateBase.doCompare(ValueComparerTemplateBase.java:33)
	at org.dbunit.assertion.comparer.value.ValueComparerBase.compare(ValueComparerBase.java:40)
	at org.dbunit.assertion.DbUnitAssertBase.compareData(DbUnitAssertBase.java:599)
	at org.dbunit.assertion.DbUnitAssertBase.compareData(DbUnitAssertBase.java:561)
	at org.dbunit.assertion.DbUnitAssertBase.assertWithValueComparer(DbUnitAssertBase.java:439)
	at org.dbunit.assertion.DbUnitAssert.assertEquals(DbUnitAssert.java:300)
	at org.dbunit.assertion.DbUnitAssert.assertEquals(DbUnitAssert.java:268)
	at org.dbunit.assertion.DbUnitAssert.assertEquals(DbUnitAssert.java:227)
	at org.dbunit.assertion.DbUnitAssert.assertEqualsIgnoreCols(DbUnitAssert.java:103)
	at com.github.database.rider.core.assertion.DataSetAssertion.assertEqualsIgnoreCols(DataSetAssertion.java:18)
	at com.github.database.rider.core.dataset.DataSetExecutorImpl.compareCurrentDataSetWith(DataSetExecutorImpl.java:793)
```

**Comment by aklemp** [comment]
Yes, I also tried with Maven exclusions and noticed this problem a bit later. Currently investigating the cause because I need a new feature of dbunit 2.6.0.

**Comment by aklemp** [comment]
It seems like dbunit is using the new Value Comparer (see also #113) which conflicts with the regex approach of database-rider.
Maybe the regex comparison should be a value comparer as well. But then the columns have to be specified that might contain regex.

**Comment by rmpestano** [comment]
Yes, I'll release v15.1 with the fix for #112 and then we can work on DBUnit upgrade for 1.6.0.

**Comment by rmpestano** [comment]
Starting with just released v`1.1.9.0`, DBRider is also compatible with DBUnit 2.6 or above. I also tested in a project using dbunit 2.5.3 without issues.
